json-cache/service-worker.js

```diff
@@ -0,0 +1,53 @@
+// [Working example](/serviceworker-cookbook/json-cache/).
+
+var CACHE_NAME = 'dependencies-cache';
+
+self.addEventListener('install', function(event) {
+  // Perform the install step:
```

Comment:
Remember you can use markdown inside docco so you could replace `-` with `*`. ;)